PCbuild/build.bat

```diff
@@ -115,7 +115,8 @@ if exist "%GIT%" set GITProperty=/p:GIT="%GIT%"
 if not exist "%GIT%" echo Cannot find Git on PATH & set GITProperty=
 
 rem Setup the environment
-call "%dir%env.bat" %vs_platf% >nul
```

Comment:
Ah, I see what you mean about keeping the warning in there. I'll bring that back